source/geometry/IntBox2D.ooc

```diff
@@ -58,7 +58,7 @@ IntBox2D: cover {
 	pad: func (left, right, top, bottom: Int) -> This {
 		This new(IntPoint2D new(this left - left, this top - top), IntVector2D new(this width + left + right, this height + top + bottom))
 	}
-	pad: func ~fromFloat (pad: Int) -> This { this pad(pad, pad, pad, pad) }
```

Comment:
`Blame David Hesselbom ( 28 months ago )`
wow, it's hard to find an older bug than that